src/datadoc/backend/datadoc_metadata.py

```diff
@@ -104,6 +106,7 @@ def extract_metadata_from_files(self) -> None:
         elif self.dataset is not None:
             self.extract_metadata_from_dataset(self.dataset)
             self.meta.dataset.id = uuid.uuid4()
+            self.meta.dataset.assessment = Assessment.PROTECTED
```

Comment:
Does this not override the logic on L190-199?

Reply:
This line is removed